draft-ietf-dnsop-serve-stale.md

```diff
@@ -364,8 +355,9 @@ NXDomain for a previously existing name might well be an error, it is
 not handled that way because there is no effective way to distinguish
 operator intent for legitimate cases versus error cases.
 
-During discussion in dnsop it was suggested that Refused from all
-authorities should be treated, from a serve-stale perspective, as
+During discussion in the IETF, it was suggested that if all auhorities
```

Comment:
"During discussion in the IETF": Since the draft is in last call, suggest we drop the phrase. Also reword first sentence for brevity (since the MAY recommendation is at the end of the paragraph) and clarity.

For a resource record if all authorities return responses with RCODE of Refused, it may be an explicit signal to take down the zone from servers that still have the zone's delegation pointed to them. Refused, ...

Reply:
WG discussion is part of IETF discussion. I assume that the document will go through WG Last Call and IETF Last Call, and that the wording should reflect both of those. Starting the sentence with "It was suggested..." will leave the reader with the question of when and by whom.

Reply:
Keep the "During discussion in the IETF" phrase but please apply the rest of the rewording.